test/cypress/integration/12-contributionFlow.donate.test.js

```diff
@@ -38,7 +38,8 @@ describe('Contribution Flow: Donate', () => {
       cy.contains('[data-cy="select-option"]', 'Monthly').click();
       cy.tick(1000); // Update details is debounced, we need to tick the clock to trigger update
       cy.contains('.step-details', '$1,337.00 per month');
-      cy.contains('Next charge: Jun 1, 2042');
```

Comment:
Can you make sure there is a test when donating before the 15th?

Reply:
The test in https://github.com/opencollective/opencollective-frontend/pull/2703/files#diff-2
checks for that